### PR TITLE
Modifcations for codebook

### PR DESCRIPTION
**Pull request by technosaby**

### CODE REVIEW

File: tagging_audio_effects/tools/codebook_generator.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+"""
+import csv
+
+if __name__ == '__main__':
+    YAMNET_CLASS_MAP_FILE = "../models/assets/yamnet_class_map.csv"
+    OUTPUT_FILE = "../codebook/codebook_yamnet_1.0.csv"
+    print("Generating codebook from ", YAMNET_CLASS_MAP_FILE)
+    input_file = open(YAMNET_CLASS_MAP_FILE, 'r')
+    output_file = open(OUTPUT_FILE, 'w')
+    data = csv.reader(input_file)
```
Comment (brucearctor) [inline tagging_audio_effects/tools/codebook_generator.py:12]: Are you sure you want to use csv.reader and writer?

Ex: https://stackoverflow.com/a/60561297 ( for writer ) ... not required, but as shared, pandas is good to know [ and handy/often-quicker to use ]

File: tagging_audio_effects/tools/codebook_generator.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    writer = csv.writer(output_file)
+
+    for line in data:
+        line.pop(1)
+        line = [value.replace(',', ' |') for value in line]
```
Comment (brucearctor) [inline tagging_audio_effects/tools/codebook_generator.py:17]: well, I guess actually for this ... https://stackoverflow.com/a/60561297

File: tagging_audio_effects/tools/codebook_generator.py (hunk 1 above)
Comment (technosaby) [reply tagging_audio_effects/tools/codebook_generator.py:12]: I wasn't trying to convert a comma delimiter to a pipe, rather keep it in csv but only change the ones with tag.
Like from (Chirp, tweet) to Chirp | tweet. 
I know about pandas and used in other tools here as well, but as it was so simple use case I did not use it. If you say I will use pandas.

Comment (brucearctor) [reply tagging_audio_effects/tools/codebook_generator.py:12]: Pandas is not strict requirement

**Comment by technosaby** [comment]
@brucearctor Just to understand clearly =, is there anything which I need to address here on this PR or you are good with this?

**Comment by brucearctor** [comment]
LGTM

**Pull request merged**